Source/Scenes/Overworld.cs

```diff
@@ -483,25 +522,56 @@ public override void Render(Target target)
 						at.X -= width2 + 8 * Game.RelativeScale;
 						UI.Prompt(batch, Controls.Pause, Loc.Str("OptionsTitle"), at, out _, 1.0f);
 					}
-
-					// show version number on Overworld as well
-					UI.Text(batch, Game.VersionString, bounds.BottomLeft + new Vec2(4, -4) * Game.RelativeScale, new Vec2(0, 1), Color.CornflowerBlue * 0.75f);
-					UI.Text(batch, Game.LoaderVersion, bounds.BottomLeft + new Vec2(4, -24) * Game.RelativeScale, new Vec2(0, 1), new Color(12326399) * 0.75f);
-					if (ModLoader.FailedToLoadMods.Any())
-					{
-						UI.Text(batch, string.Format(Loc.Str("FailedToLoadMods"), ModLoader.FailedToLoadMods.Count), bounds.BottomLeft + new Vec2(4, -44) * Game.RelativeScale, new Vec2(0, 1), Color.Red * 0.75f);
-					}
 				}
 
-				if (cameraCloseUpEase > 0)
+				if (ModLoader.FailedToLoadMods.Any())
```

Comment:
I feel like the failed to load mods check and display should maybe be moved down into the same if check block where you show the version strings, since it displays in kind of the same place on screen, and probably should follow the same show/hide logic.

Reply:
Resolved in 50daf0d